public/app/percona/backup/Backup.messages.ts

```diff
@@ -3,6 +3,72 @@ export const Messages = {
     inventory: 'Backup Inventory',
     scheduled: 'Scheduled Backups',
     locations: 'Storage Locations',
+    restore: 'Restore History',
   },
+  add: 'Add',
   backupManagement: 'Backup Management',
+  backupInventory: {
+    table: {
+      noData: 'No Backups found',
+      columns: {
+        name: 'Backup name',
+        created: 'Created',
+        location: 'Location',
+        vendor: 'Vendor',
+        status: 'Status',
+        actions: 'Actions',
+      },
+      status: {
+        invalid: 'Invalid',
+        pending: 'Pending',
+        inProgress: 'In progress',
+        paused: 'Paused',
+        success: 'Success',
+        error: 'Error',
+      },
+      dataModel: {
+        invalid: 'Invalid',
+        physical: 'Physical',
+        logical: 'Logical',
+      },
+      actions: 'Actions',
+    },
+  },
+  restoreHistory: {
+    table: {
+      noData: 'No Restores found',
+      columns: {
+        started: 'Started At',
```

Comment:
`Started At`, `No Restores found` and `In progress` are not consistent with what words start with capital letter and what not. I would keep everything lower case except the first word in this case.

Reply:
good point 👍